src/rust/engine/process_execution/src/remote.rs

```diff
@@ -387,7 +387,9 @@ impl super::CommandRunner for CommandRunner {
                     "Exceeded timeout of {:?} ({:?} for the process and {:?} for remoting buffer time) with {:?} for operation {}, {}",
                     total_timeout, timeout, self.queue_buffer_time, elapsed, operation_name, description
                   )),
+                  stdout_digest: hashing::EMPTY_DIGEST,
```

Comment:
This is a bug, since the content and the digest mismatch. More reason not to hold both on the type.